client/my-sites/importer/section-import.jsx

```diff
@@ -261,9 +260,15 @@ class SectionImport extends Component {
 	};
 
 	renderImportersList() {
+		const { translate } = this.props;
+		const isSpecificImporter = ! isEmpty( this.state.importers );
+		const sectionHeaderLabel = isSpecificImporter
+			? translate( 'Importing content from:' )
```

Comment:
This could use something like a `{ comment: 'This text appears above the icon of another service (e.g. Wix, Squarespace) indicating that the process of importing the user's data from that service is ongoing' }`

Sentence fragments can be a bit tricky.